src/Components/Server/src/Circuits/ICircuitHandleRegistry.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.Lifetime;
+using Microsoft.AspNetCore.Components.Routing;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.JSInterop;
+
+namespace Microsoft.AspNetCore.Components.Server.Circuits
+{
+    internal interface ICircuitHandleRegistry
+    {
+        public CircuitHandle GetCircuitHandle(object circuitKey);
+
+        public CircuitHost GetCircuit(object circuitKey);
+
+        public void SetCircuit(object circuitKey, CircuitHost circuitHost);
```

Comment:
TBH I didn't realise that `public` is now a valid modifier on interface members, but I see it now is :)

Since none of our other interface definitions use `public` like this, would it be preferable to remove those modifiers? Don't mind strongly.

Reply:
Also on `IServerComponentDeserializer`